### PR TITLE
Add how-to on Python to MATLAB interface

### PR DESCRIPTION
**Pull request by jwbuurlage**

### CODE REVIEW

File: how-to-guide.org [hunk 1]
```diff
@@ -9,6 +9,43 @@
 *** TODO Auto-format your code
 **** PEP8
 **** [[https://github.com/google/yapf][yapf]]
+*** Call MATLAB from Python
+\textsc{Jan-Willem Buurlage}\\
```
Comment (ahendriksen) [inline how-to-guide.org]: Does textsc work for the HTML export?

**Review by ahendriksen — COMMENTED** [review]
Looks good! Can you check the textsc?

**Comment by wjp** [comment]
Ah, very nice. I quickly tested the instructions, and they work, except that the sample call to `triarea` misses a second argument.

**Comment by jwbuurlage** [comment]
I added a second argument, and removed the `\textsc`.

How do you feel about adding section authors? It might motivate people if they get some credits, and it may be useful information to readers as they can initially ask the author for more details if necessary.

**Comment by ahendriksen** [comment]
> How do you feel about adding section authors? It might motivate  people if they get some credits, and it may be useful information to  readers as they can initially ask the author for more details if  necessary.

I think it is a good idea for the reasons you cite.

**Comment by jwbuurlage** [comment]
Ok, I'll make a separate PR adding authors for every current section.

For me, this how-to is OK to merge.

**Comment by adriaangraas** [comment]
Looks all good, authors sounds good. Compile docs?

**Review by adriaangraas — APPROVED** [review]

**Comment by jwbuurlage** [comment]
Added everything in this PR: authors + compiled docs.

**Pull request merged**